tsl/src/reorder.c

```diff
@@ -794,6 +794,7 @@ copy_heap_data(Oid OIDNewHeap, Oid OIDOldHeap, Oid OIDOldIndex, bool verbose,
 			case HEAPTUPLE_RECENTLY_DEAD:
 				tups_recently_dead += 1;
 				/* fall through */
```

Comment:
renaming this comment to "fallthrough" could also help

Reply:
it seems like a peculiarity of clang 10, look at this (unfortunate) thread https://bugs.llvm.org/show_bug.cgi?id=43465.  in newer version and gcc the existing comment is sufficient.